database/controllers/auth.js

```diff
@@ -0,0 +1,22 @@
+const client = require('../index');
+
+const find = function (username, password) {
+  return client.query(`
+    SELECT *
+    FROM auth
+    WHERE username = '${username}'
+    AND password = '${password}'
+  `)
+    .then((data) => data.rows)
```

Comment:
Instead of all rows, we maybe want to return only username and type (fan/artist).